ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -137,289 +149,351 @@ Requirements are located here: [avd-requirements](../../README.md#Requirements)
 - Available features  and variables may vary by platforms, refer to documentation on arista.com for specifics.
 - All values are optional.
 
-### Terminal Settings
+### Management
+
+#### Management Interfaces
 
 ```yaml
-terminal:
-  length: < 0-32767 >
-  width: < 0-32767 >
+management_interfaces:
+  < Management_interface_1 >:
+    description: < description >
+    shutdown: < true | false >
+    vrf: < vrf_name >
+    ip_address: < IPv4_address/Mask >
+    ipv6_enable: < true | false >
+    ipv6_address: < IPv6_address/Mask >
+    gateway: <IPv4 address of gateway>
```

Comment:
I cannot see this being read in any template.

Reply:
Remove it from documentation with commit 8c97a1a